### ISSUE TITLE
Line break introduced in the middle of `cross join`

### ISSUE THREAD

**Issue by rdeese** [tconbeer/sqlfmt#110]
Hi, wanted to lead off by saying how excited we are by this tool. Thanks for your work on it. Now, to the issue at hand.

sqlfmt 0.4.2 formats this query with a line break between "cross" and "join":
```
select foo.*, bar.*
from {{ ref('foo_foo_foo') }} as foo cross
join {{ ref('bar_bar_bar') }} as bar
```
I would prefer to have `cross join` together on the new line, like so:
```
select foo.*, bar.*
from {{ ref('foo_foo_foo') }} as foo
cross join {{ ref('bar_bar_bar') }} as bar
```
Feel free to close the issue if this is intended behavior, but I found it surprising and suspect it might be a bug.

**Comment by tconbeer** [comment]
Thanks for the report, @rdeese ! I'm excited too 😀 

What you're seeing is definitely not the intended behavior. My guess is that we're not parsing that token correctly. Should be an easy fix if that's the case. I'll keep you updated here

**Comment by tconbeer** [comment]
Yep, this is easy. Should be fixed in 0.5.0 this week

**Comment by tconbeer** [comment]
0.5.0 has been released! Please upgrade for the patch.
